cmd/users/main.go

```diff
@@ -178,7 +178,8 @@ func connectToDB(dbConfig postgres.Config, logger logger.Logger) *sqlx.DB {
 }
 
 func newService(db *sqlx.DB, tracer opentracing.Tracer, secret string, logger logger.Logger) users.Service {
-	repo := tracing.UserRepositoryMiddleware(postgres.New(db), tracer)
+	database := postgres.NewDatabase(db)
+	repo := tracing.UserRepositoryMiddleware(postgres.New(database), tracer)
```

Comment:
Why this New here?

Reply:
The UserReposityoryMiddleware creates the initial span and passes it via context when it calls the db interface.  It needs access to the tracer in order to do so

Reply:
My question was more about code structure. Looks like you are passing a `dbTracer` for Things and Channels but not here. Thank you for the answer in any case. I'll resolve this one for one and we will see later.